<?xml version="1.0"?>
<launch>
    <arg name="calibrated_marker_path" default="$(find smartworkcell)/config/calibrated_marker.yaml"/>
    
    <node name="camera_calibration_node" pkg="smartworkcell" type="CameraRobotCalibration.py"  output="screen" respawn="false" respawn_delay="0" args="">
        <rosparam command="load" file="$(find smartworkcell)/config/camera_calibration_params.yaml"/>
        <param name="calibrated_marker_path" value="$(arg calibrated_marker_path)"/>
    
    </node>
</launch>